Source/ARTBaseMessage.h

```diff
@@ -29,6 +30,10 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (strong, nonatomic, nullable) id data;
 
+/// The event name, if available
+@property (nullable, readwrite, strong, nonatomic) NSString *name;
```

Comment:
I don't understand why `name` is moved. Is the idea to include `name` in `PresenceMessage` ? That shouldn't happen.

Reply:
So is it not easy to do:

```
ARTMessage.messageSize() {
  return super.messageSize() + name.length;
}
```

?

Reply:
That's one way to do it. I was caught by other aspects of this PR and I didn't think of it :)
Issue here: https://github.com/ably/ably-ios/issues/762